rai/node/rpc.cpp

```diff
@@ -3126,10 +3126,24 @@ void rai::rpc_handler::unchecked_keys ()
 
 void rai::rpc_handler::version ()
 {
+	bool patch (false);
+	boost::optional<bool> patch_optional (request.get_optional<bool> ("patch"));
+	if (patch_optional.is_initialized ())
+	{
+		patch = patch_optional.get ();
+	}
+
```

Comment:
All of that can be reduced to `bool patch = request.get<bool> ("patch", false);`

Reply:
I agree I think this would be a good idea at least here @clemahieu thoughts?